install_files/ansible-base/roles/app/tasks/initialize_securedrop_app.yml

```diff
@@ -180,3 +180,11 @@
   tags:
     - logo
     - securedrop_config
+
+- name: Add DEFAULT_LOCALE to config.py if missing.
+  shell: >
+    grep -q DEFAULT_LOCALE {{ securedrop_code }}/config.py ||
+    echo "DEFAULT_LOCALE = 'en_US'" | tee -a {{ securedrop_code }}/config.py
+  when: db.stat.exists
+  tags:
+    - securedrop_config
```

Comment:
Use `lineinfile` rather than `shell`—that's what it's for!

```
- name: Add DEFAULT_LOCALE to config.py if missing.
  lineinfile:
    line: "DEFAULT_LOCALE = 'en_US'"
    regexp: "^DEFAULT_LOCALE"
  when: db.stat.exists
```

We conservatively do _not_ modify config.py for the multiline declarations, e.g. with conditional logic. Even the changes here will require Admin intervention, but general config problems are tracked in #1966.

Reply:
Using a default role var would look like:

```
- name: Add DEFAULT_LOCALE to config.py if missing.
  lineinfile:
    line: "DEFAULT_LOCALE = '{{ securedrop_default_locale }}'"
    regexp: "^DEFAULT_LOCALE"
  when: db.stat.exists
```